db/util_testing.go

```diff
@@ -644,3 +644,25 @@ func DefaultMutateInOpts() *sgbucket.MutateInOptions {
 		MacroExpansion: macroExpandSpec(base.SyncXattrName),
 	}
 }
+
+func constructDocumentFromBody(docID string, body Body) (newDoc *Document) {
```

Comment:
This is doing a lot of work that's not actually required by the new test usage (which doesn't include any of these reserved properties in the body).    In particular this looks like it's designed to handle document body properties in the way the 1.x REST API does, but we don't currently implement REST API support for HLVs. 

Any reason to use this approach, as opposed to a helper function like createTestDocument(docID string, revID string, body Body, deleted bool, ...)?